### ISSUE TITLE
Grunt fails to build some(?) files

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks_homepage#6]
Now that I installed grunt in the folder correctly, there are files missing from the build(s)?:

```
matt@MacBook-Pro-71 taxonworks_homepage (develop)$ grunt build
Running "clean:build" (clean) task
>> 1 path cleaned.

Running "buildHTMLWithoutCritical:dev" (buildHTMLWithoutCritical) task

Running "buildHTMLWithoutCritical:production" (buildHTMLWithoutCritical) task

Running "stylus:dev" (stylus) task
>> 1 file created.

Running "stylus:production" (stylus) task
>> 1 file created.

Running "concat:dev" (concat) task

Running "concat:productionBeforeTranspile" (concat) task

Running "babel:production" (babel) task

Running "concat:productionAfterTranspile" (concat) task

Running "concat:dev" (concat) task

Running "concat:productionBeforeTranspile" (concat) task

Running "concat:productionAfterTranspile" (concat) task

Running "copy:dev" (copy) task
Copied 26 files

Running "copy:production" (copy) task
Copied 26 files

Running "critical:production" (critical) task
>> Destination (build/production/index.html) not written because src files were empty.
matt@MacBook-Pro-71 taxonworks_homepage (develop)$ 

```

**Comment by BrianWalters** [comment]
Found the problem. Grunt is silently dying when processing the scope config. I wrapped the `scope.process` call in the Gruntfile in a try...catch and forced a grunt failure. This way, your build will be interrupted by the error. The problem is this:

```
Fatal error: description missing on feature {"name":"Specimen and collecting events","status":"complete","description":null,"whenComplete":"now","features":Array(2)}
```

We have some validation on the scope data to make sure some properties are complete and the right enums are used. Hope this makes sense.
